test/cpp/jit/backend_with_compiler.cpp

```diff
@@ -0,0 +1,111 @@
+#include <torch/csrc/jit/backends/backend.h>
+
+namespace torch {
+namespace jit {
+
```

Comment:
It'd be good to document up here the behaviour/contract of this backend: e.g. Implementation of a PyTorch Backend that can process, compile and execute TorchScript Modules composed of 'add' and 'sub' operators between constants and expect 2 inputs of type Tensor of blah... which are added or subtracted to blah blah

You can punt this until you add support for constants.

But basically it'd be good if this backend, however simple (it really doesn't need to be fancy), implemented some well defined algorithm that used all the bits the preprocessed/compiled Module rather than just hardcoding some values, and ignoring others.

Reply:
Good suggestion! I'll add more comments, and discuss with you on defined algorithms offline.

Reply:
Now I get your point of "well defined algorithm". I'll add the support for constant.